CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [1.11.0] - 2024-10-24
+
+### Added
+
+- Support for forced rotation and revocation (<https://github.com/orgs/spiffe/projects/21>)!!!
+- New EJBCA UpstreamAuthority plugin for SPIRE Server (#5378)
+- Support for variables in templates contained in the config file (#5576)
+- Support for the configuration validation RPC on all built-in plugins (#5303)
+- Improved logging when built-in plugins panic (#5476)
+- Improved CPU and memory resource usage for concurrent Kubernetes Workload attestation (#5408)
+- Documentation additions and improvements (#5589, #5588, #5499, #5433, #5430, #5269)
+
+### Changed
+
+- SPIRE Agent LRU identity cache is now unconditionally enabled. The LRU size can be controlled via the `x509_svid_cache_max_size` configuration option. (#5383, #5531)
+- Entry API RPCs return per-entry InvalidArgument status when creating/updating malformed entries (#5506)
+- Reverted a previous change to the agent health check in 1.10.2 that broke some deployments (#5461)
```

Comment:
We released 1.10.3 with this fix, so it's already part of the changelog.

Reply:
Oh weird. This fix was tagged against the milestone. I'll remove.

Reply:
I'll also update 5461 with the correct milestone.

Reply:
We had two separate PRs, one for 1.10.3 (#5459) and one for the main branch (#5461). #5461 Should have the 1.11.0 milestone.

Reply:
OH right. thanks for the reminder.